packages/PackageBuilder/src/Strings/StringFormatConverter.php

```diff
@@ -8,11 +8,9 @@ final class StringFormatConverter
 {
     public function underscoreToCamelCase(string $value): string
     {
-        $value = str_replace(' ', '', ucwords(str_replace('_', ' ', $value)));
+        $value = str_replace(' ', '', ucwords(str_replace(['-', '_'], ' ', $value)));
```

Comment:
This is different behavior. Why?

Reply:
I actually copy from laravel, it handles dash.  I was also kind of worried about breaking change. I'll remove the dash replacement.

Reply:
On a second thought. 
the old one `Str::camel()` will replace ['-', '_'] with whitespace.
I check the whole repository and I can find two places using `underscoreToCamelCase` in StringFormatConverter.php and StringFormatConverterTest.php
I guess, it's safe to change this behavior?

Reply:
if yes, should I rename the method to avoid the confusion?

Reply:
I see. In that case laravel lied to us :+1: 

> if yes, should I rename the method to avoid the confusion?

:+1: That would be annoying bug to look for, if someone else would try to keep `-` in their code

Reply:
Also add test with "_some-value " input and expected output, just to be sure